### DISCUSSION TITLE
Copilot Studio_Adaptive card issue_Property cannot be Null

### DISCUSSION THREAD

**Discussion by pu3aiesya** [microsoft/AdaptiveCards#8834] [Q&A]
Hi all, below is my JSON for my adaptive card in copilot studio.
I got an error saying that : Property cannot be null?
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "TextBlock",
            "size": "Medium",
            "weight": "Bolder",
            "text": "FREQUENTLY ASKED QUESTION [FAQ]",
            "horizontalAlignment": "Center"
        },
        {
            "type": "Image",
            "url": "https://static.wixstatic.com/media/6e51ea_b20110d8e8c04018813c6d859273c14f~mv2.gif"
        },
        {
            "type": "TextBlock",
            "text": "Here are some available topics which you can dive in and have a look first.",
            "wrap": true,
            "separator": true
        },
        {
            "type": "Input.ChoiceSet",
            "choices": [
                {
                    "title": "Access",
                    "value": "Access Issue"
                },
                {
                    "title": "LPN",
                    "value": "Missing LPN 2"
                },
                {
                    "title": "Status",
                    "value": "Change Request Status"
                },
                {
                    "title": "FAQ",
                    "value": "FAQ"
                }
            ],
            "placeholder": "Select an option"
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.5",
    "actions": [
        {
            "type": "Action.Submit",
            "title": "Submit",
            "iconUrl": "https://pngimg.com/d/buttons_PNG151.png"
        }
    ]
}

![image](https://github.com/microsoft/AdaptiveCards/assets/161137201/419783d7-2ef3-4e38-b317-94402933949a)

**Comment by jwoo-msft** [comment]
@pu3aiesya,

Can you provide more information about copilot studio?

It does appear that it's specific to copilot studio as this card works in our [designer](https://adaptivecards.io/designer/).
